frontend/public/locales/en/translation.json

```diff
@@ -2,6 +2,7 @@
     " No status: ": " No status: ",
     " With violations: ": " With violations: ",
     " Without violations: ": " Without violations: ",
+    "({{0}} of {{0}})": "({{0}} of {{0}})",
```

Comment:
This appears in `frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/components/ChannelControl.js` but it seems it wasn't added to translation until I ran the script for my branch.

Reply:
Please take note @jeswanke :) i18next-parser should be running during pre-commit, or you can run `npm run i18n` manually. But thank you for externalizing.

Reply:
oops--lots of moving parts in. this one--thanks for tha catch